dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -908,6 +908,6 @@ portalRouter.route('/gef/mandatory-criteria/version/:version').get(mandatoryCrit
  *         description: Server conflict
  */
 portalRouter.route('/utilisation-reports').post(utilisationReportUpload.postUtilisationReportData);
-portalRouter.route('/previous-reports/:bankId').get(validation.bankIdValidation, handleValidationResult, previousReports.getUtilisationReports);
+portalRouter.route('/banks/:bankId/utilisation-reports').get(validation.bankIdValidation, handleValidationResult, previousReports.getUtilisationReports);
```

Comment:
Should this go in `bank-routes.js` now? If so we should change also it to `'/bank/:bankId/utilisation-reports'` (singular `bank`).

Reply:
While some of this is preference on single vs plural and the preference in this codebase is plural, I also think plural nouns are better for this kind of thing. We are picking a bank from a collection of banks and getting utilisation-reports from that bank.

For bank-routes now I considered this and was struggling with getting my head round the portal and tfm sets of routes which exist due to the different collections, but if they also just contained 'portal related' routes in an annoying sort of way. I think I'm happy to move it to bank-routes and it's a good suggestion just thinking out loud :D

Reply:
Oh no, under /bank its /bank instead of /banks :( I should probably fix this but might raise a separate tech debt ticket to refactor (we did some pluralisation of nouns refactoring not that long ago). Looks like for now it'll have to just be /bank, sad times

Reply:
I noticed inyour PR the get all banks endpoints /bank

Reply:
If we do refactor `bank-routes.js`to use `/banks` instead of `/bank` we should wait until this PR has gone through (to remove the duplicate bank routes from `portal-routes.js`) as `portal-routes.js` is currently already using `/banks`.

Reply:
Yep can do it separately as will need to run it by DTFS team first anyway